app/models/dossier.rb

```diff
@@ -294,6 +294,8 @@ def classer_sans_suite(motivation: nil, instructeur: nil, processed_at: Time.zon
       champs: [:type_de_champ, piece_justificative_file_attachments: :blob]
     ])
   }
+
+  scope :brouillons_recently_updated, -> { not_archived.updated_since(2.days.ago).state_brouillon.order_by_updated_at }
```

Comment:
```suggestion
  scope :brouillons_recently_updated, -> { updated_since(2.days.ago).state_brouillon.order_by_updated_at }
```

un brouillon ne peut pas être archivé